pallets/automation-time/src/benchmarking.rs

```diff
@@ -173,6 +210,17 @@ benchmarks! {
 		let provided_id: Vec<u8> = vec![(T::MaxTasksPerSlot::get()/256).try_into().unwrap(), (T::MaxTasksPerSlot::get()%256).try_into().unwrap()];
 	}: schedule_native_transfer_task(RawOrigin::Signed(caller), provided_id, times, recipient, transfer_amount)
 
+	schedule_auto_compound_delegated_stake_task_full {
```

Comment:
My first time doing benchmarking so worth a good look here to make sure I did it right.